### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by krtrinkunas**
add class GameLogic with some game logic

### CODE REVIEW

File: EncounterMe/EncounterMe/Classes/Location.cs [hunk 1]
```diff
@@ -23,7 +22,7 @@ public Location(int ID, String Name, double Latitude, double Longtitude)
             this.Longtitude = (float) Longtitude;
         }
 
-        public float distanceToUser(double lat, double lon)
+        public float distanceToUser(float lat, float lon)
         {
             return (float) (circumference * Math.Acos(Math.Sin(this.Latitude * Math.PI / 180.00) * Math.Sin(lat * Math.PI / 180.00) + Math.Cos(this.Latitude * Math.PI / 180.00) * Math.Cos(lat * Math.PI / 180.00) * Math.Cos((this.Longtitude - lon) * Math.PI / 180)));
```
Comment (DominykasVi) [inline EncounterMe/EncounterMe/Classes/Location.cs]: Put in separate lines. Not one big line.

**Review by DominykasVi — APPROVED** [review]
Everything else looks good.

**Review by AdamBeresnev — APPROVED** [review]
I see no issues

File: EncounterMe/EncounterMe/Functions/GameLogic.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                }
+            }
+            Random random = new Random();
+            int index = random.Next(locationsSortedByDistance.Count);
+            try
+            {
+                Location location =  locationsSortedByDistance[index];
+                return location;
+            }
+            catch (ArgumentOutOfRangeException e)
```
Comment (EmilijaSvirsk) [inline EncounterMe/EncounterMe/Functions/GameLogic.cs:26]: Will need an exception handler in the future. Other than that everything is good.

**Review by EmilijaSvirsk — APPROVED** [review]

**Pull request merged**